src/site/xdoc/server/config-cassandra.xml

```diff
@@ -76,22 +76,27 @@
         # cassandra.pooling.timeout=5000
         ## In seconds.
         # cassandra.pooling.heartbeat.timeout=30
+        # cassandra.pooling.max.queue.size=256
 
         <dt><strong>cassandra.pooling.local.max.connections</strong></dt>
         <dd>Optional.
-          If specified defines the Cassandra <a href="http://docs.datastax.com/en/developer/java-driver/2.1/manual/pooling/">maximum number of connections to hosts</a> (remote and local)</dd>
+          If specified defines the Cassandra <a href="https://docs.datastax.com/en/developer/java-driver/3.5/manual/pooling/">maximum number of connections to hosts</a> (remote and local).</dd>
 
         <dt><strong>cassandra.pooling.local.max.requests</strong></dt>
         <dd>Optional.
-          If specified defines the Cassandra <a href="http://docs.datastax.com/en/developer/java-driver/2.1/manual/pooling/">maximum number of concurrent requests per connection</a></dd>
+          If specified defines the Cassandra <a href="https://docs.datastax.com/en/developer/java-driver/3.5/manual/pooling/">maximum number of concurrent requests per connection</a>.</dd>
 
         <dt><strong>cassandra.pooling.timeout</strong></dt>
         <dd>Optional.
-          If specified defines the Cassandra <a href="http://docs.datastax.com/en/developer/java-driver/2.1/manual/pooling/">Specify timeout while not locationg pool</a>. Should be higher than sockets timeout.</dd>
+          If specified defines the Cassandra <a href="https://docs.datastax.com/en/developer/java-driver/3.5/manual/pooling/#acquisition-queue">timeout for waiting in the pool queue</a>. Should be higher than sockets timeout.</dd>
 
         <dt><strong>cassandra.pooling.heartbeat.timeout</strong></dt>
         <dd>Optional.
-          If specified defines the Cassandra <a href="http://docs.datastax.com/en/developer/java-driver/2.1/manual/pooling/">Specify heartbeat timeout</a></dd>
+          If specified defines the Cassandra <a href="https://docs.datastax.com/en/developer/java-driver/3.5/manual/pooling/#heartbeat">heartbeat timeout</a>.</dd>
```

Comment:
Can you specify that the default is 256?

Reply:
It should be 30 for **heartbreak** timeout